setup-earthly/libs/setup-earthly.js

```diff
@@ -0,0 +1,36 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const child_process = require('child_process');
```

Comment:
Should you be using imports here?

Reply:
I have no idea, I'm no node-expert; it's mostly various bits of stackoverflow+github copy and pasting :joy:

Reply:
Might be a good idea - see https://github.com/airbnb/javascript#modules